BLSQ/iaso
# IA-1947 handle bulk import with task worker

The bulk create user has been moved as a task. To do it I moved the code that actually create the users out of the view and made it a task. Tests are covering the user creation part. 

Related JIRA tickets : IA-1873

## Self proofreading checklist


## How to test

make a csv file to create users. Check the files from the tests to find how to do it. 
go to /api/bulkcreateuser/ and upload your file. 
Go to task and check if the task is created. 


For the user creation itself,  you must run it from iaso bash using :

` bulk_create_users_task.__wrapped__(uploaded_file, user, False)`

uploaded_file is your csv file mock as an upload.
user is your logged in user
False is to prevent triggering task log when not performing it as a task.


Dont forget to wrap it to dont trigger the task. 

The file must be a mock of an uploade file so you must do it this way : 

```
file = open("path_to_your_test_file")

request = RequestFactory().post("/upload", {"file": file})
uploaded_file = request.FILES["file"]
```
